templates/etc/init.d/deb/td-agent

```diff
@@ -118,7 +118,7 @@ do_start() {
   if kill_by_file -0 "${TD_AGENT_PID_FILE}"; then
     return 1
   else
-    GOOGLE_LOGGING_ENABLE=$(curl --silent --connect-timeout 1 -f -H "Metadata-Flavor: Google" http://169.254.169.254/computeMetadata/v1/instance/attributes/google-logging-enable 2>/dev/null)
+    GOOGLE_LOGGING_ENABLE=$(curl --silent --connect-timeout 1 -f -H "Metadata-Flavor: Google" http://169.254.169.254/computeMetadata/v1/instance/attributes/google-logging-enable 2>/dev/null || /bin/true)
```

Comment:
A few questions. Why use /bin/true, instead of just "true"?
Could we just `|| echo ""` instead to be clear that we're simulating an empty response?
Same questions apply to the other change here.